code/game/machinery/pipe/pipe_dispenser.dm

```diff
@@ -107,31 +107,31 @@
 		qdel(W)
 		return
 	else if(istype(W, /obj/item/wrench))
-		if(unwrenched==0)
+		if(unwrenched)
+			playsound(loc, W.usesound, 50, 1)
+			to_chat(user, "<span class='notice'>You begin to fasten \the [src] to the floor...</span>")
+			if(do_after(user, 20 * W.toolspeed, target = src))
+				user.visible_message( \
```

Comment:
would you look into making this a `wrench_act()`

Reply:
Done.
Now I'm just wondering why `/obj/item/reagent_containers/food/snacks/sliceable` doesn't override `tool_act` to add a `slice_act`